src/components/pages/search-page/SearchPage.jsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react'
+import LabelledInput from '../../labelled-input/LabelledInput'
+import './SearchPage.css'
+import Button from '../../button/Button'
 
 const SearchPage = () => {
-  return <div />
+  return (
+    <>
+      <form action="" className="search-page-form">
+        <LabelledInput
+          label="ID заказа"
+          labelClassName="search-page-form-field"
+          centered
+        />
+        <LabelledInput
+          label="Телефон"
+          labelClassName="search-page-form-field"
+          centered
+        />
+        <LabelledInput
+          label="Город"
+          labelClassName="search-page-form-field"
+          centered
+        />
+        <LabelledInput
+          label="Улица"
+          labelClassName="search-page-form-field"
+          centered
+        />
+        <LabelledInput
+          label="Дом"
+          labelClassName="search-page-form-field"
+          centered
+        />
+        <Button
+          className="
+          search-page-form-field-button"
```

Comment:
давай поместим название класса на предыдущую строчку

Reply:
и button это не особо field. Давай назовем класс чуть иначе
```suggestion
          search-page-form-submit-button"
```